File: server/src/main/java/ru/vitaliy/petrov/server/forms/requests/userprofile/DriverLicenseUpdateRequest.java
package ru.vitaliy.petrov.server.forms.requests.userprofile;

import lombok.Data;

import javax.validation.constraints.NotEmpty;
import javax.validation.constraints.NotNull;
import java.sql.Date;

@Data
public class DriverLicenseUpdateRequest {

    @NotNull
    @NotEmpty
    private final String updatedDriverLicenseNumber;

    @NotNull
    @NotEmpty
    private final String updatedDriverLicenseCategory;

    @NotNull
    private final Date updatedDriverLicenseDateOfIssue;
}

File: client/app/src/main/java/com/petrov/vitaliy/caraccidentapp/domain/models/requests/user/DriverLicenseUpdateRequest.kt
package com.petrov.vitaliy.caraccidentapp.domain.models.requests.user

import java.sql.Date

data class DriverLicenseUpdateRequest(
    val updatedDriverLicenseNumber: String,
    val updatedDriverLicenseCategory: String,
    val updatedDriverLicenseDateOfIssue: Date
)
